src/main/java/eu/dissco/backend/repository/AnnotationRepository.java

```diff
@@ -39,13 +40,13 @@ public List<Annotation> getAnnotations(int pageNumber, int pageSize) {
         .limit(pageSizePlusOne).offset(offset).fetch(this::mapToAnnotation);
   }
 
-  public int getAnnotationForUser(String id, String userId) {
-    return context.select(ANNOTATION.ID)
+  public Optional<Annotation> getAnnotationForUser(String id, String userId) {
```

Comment:
Maybe rename method to getActiveAnnotationForUser (unrelevant to this PR I know)